profile/README.md

```diff
@@ -0,0 +1,12 @@
+Welcome to Pulumi open source! :purple_heart: :cloud:
+
+We are glad that you are here! We love and appreciate the Pulumi community.
+We use [Pulumi Slack](https://slack.pulumi.com/)
+and [Pulumi GitHub Discussions](https://github.com/pulumi/pulumi/discussions) for community conversations; use whichever you like.
+
+Want to contribute? Check out the [Pulumi Contributing Guide](https://github.com/pulumi/pulumi/blob/master/CONTRIBUTING.md).
```

Comment:
we should also include the pulumiverse here
https://github.com/pulumiverse

Reply:
in what context? like so they go to that org and contribute there instead?

Reply:
I wouldn’t add this